sentry_sdk/integrations/httpx.py

```diff
@@ -139,6 +162,88 @@ async def send(self, request, **kwargs):
             span.set_http_status(rv.status_code)
             span.set_data("reason", rv.reason_phrase)
 
+            _capture_graphql_errors(hub, request, rv)
+
             return rv
 
     AsyncClient.send = send
+
+
+def _make_request_processor(request, response):
+    # type: (Request, Response) -> EventProcessor
+    def httpx_processor(
+        event,  # type: Dict[str, Any]
+        hint,  # type: Dict[str, Tuple[type, BaseException, Any]]
+    ):
+        # type: (...) -> Dict[str, Any]
+        with capture_internal_exceptions():
+            request_info = event.setdefault("request", {})
+
+            parsed_url = parse_url(str(request.url), sanitize=False)
+            request_info["url"] = parsed_url.url
+            request_info["method"] = request.method
+            request_info["headers"] = _filter_headers(dict(request.headers))
+            request_info["query_string"] = parsed_url.query
+
+            request_content = request.read()
+            if request_content:
+                try:
+                    request_info["data"] = json.loads(request_content)
```

Comment:
Guard with `_should_send_default_pii()`. Because request bodies can probably have lots of personal data in them.